File: Desenvolvimento/server/src/ApplicationCore/Entities/Estado.cs
using ApplicationCore.Enums;
using ApplicationCore.Interfaces.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationCore.Entities
{
    public class Estado : IBaseEntity<long>
    {
        public long Id { get; set; }
        public string Nome { get; set; }
        public string Sigla { get; set; }

        public long PaisId { get; set; }
        public virtual Pais Pais { get; set; }

        public virtual ICollection<Municipio> Municipios { get; set; } = new List<Municipio>();
        public virtual ICollection<Logradouro> Logradouros { get; set; } = new List<Logradouro>();
    }
}

File: Desenvolvimento/client/src/@fuse/types/models/estado.ts
import { Pais } from './pais';
import { BaseModel } from '../base-model';

export class Estado extends BaseModel {
    constructor(
        public id: number,
        public nome?: string,
        public sigla?: string,
        public paisId?: number,
        public pais?: Pais
    ) { super(); }
}
